qe-tests/cypress/integration/types/types.ts

```diff
@@ -45,6 +45,7 @@ export type PlanData = {
   useExistingNetworkMapping: boolean;
   useExistingStorageMapping: boolean;
   providerData: ProviderData;
+  targetprovider: ProviderData;
```

Comment:
I guess your mistake is here, you should do this optional. Also, can you please do a camel style? `targetProvider` will be better

Reply:
That was an oversight .